__tests__/app.test.js

```diff
@@ -56,3 +56,38 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("responds with 200 and corresonding article", () => {
+    return request(app)
+      .get("/api/articles/7")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toEqual([{
```

Comment:
Instead of toEqual, objectContaining or toMatchObject would be more helpful here, as the endpoint will change later in the project. 👍